File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let lux = 50, luy = 50
    let rdx = 0, rdy = 0
    
    for (let i = 0; i < wallpaper.length; i++) {
        for (let j = 0; j < wallpaper[0].length; j++) {
            if (wallpaper[i][j] === '#') {
                lux = Math.min(lux, i)
                luy = Math.min(luy, j)
                rdx = Math.max(rdx, i + 1)
                rdy = Math.max(rdy, j + 1)
            }
        }
    }    
    return [lux, luy, rdx, rdy];
}

File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    
    lux, luy = 50, 50
    rdx, rdy = 0, 0

    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[i])):
            if wallpaper[i][j] == '#':
                if lux > i:
                    lux = i
                if luy > j:
                    luy = j
                if rdx < i + 1:
                    rdx = i + 1
                if rdy < j + 1:
                    rdy = j + 1
        
    return [lux, luy, rdx, rdy]